instrumentation/runtime/runtime.go

```diff
@@ -111,6 +111,15 @@ func (r *runtime) register() error {
 	uptime, err := r.meter.Int64ObservableCounter(
 		"runtime.uptime",
 		metric.WithUnit("ms"),
+		metric.WithDescription("Milliseconds since application was initialized. Deprecated metric. Use `process.runtime.runtime` instead."),
```

Comment:
@open-telemetry/go-approvers what do you all think about this way of deprecating/renaming a metric?
It does mean for one release we emit two metrics.

Reply:
Is there anything you would like me to change  @dmathieu